core/runtime.rs

```diff
@@ -1937,6 +2030,30 @@ impl JsRealm {
     &self.0
   }
 
+  pub(crate) fn state(
+    &self,
+    isolate: &mut v8::Isolate,
+  ) -> Rc<RefCell<ContextState>> {
+    self
+      .context()
+      .open(isolate)
+      .get_slot::<Rc<RefCell<ContextState>>>(isolate)
+      .unwrap()
+      .clone()
+  }
+
+  /// Gets the state associated to a scope's current context, and calls the
+  /// passed callback with it.
```

Comment:
What callback?

Reply:
This is stale documentation from a previous version of this method that called a callback with a `&mut ContextState` rather than returning a `Rc<RefCell<ContextState>>`.